docs/consensus/pot.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Proof-of-Time
+sidebar_position: 5
+description: Proof-of-Time consensus component
+keywords:
+    - Consensus
+    - randomness
+    - challenge
+---
+In addition to Proof-of-Space component described in previous section, Dilithium is secured by a Proof-of-Time (PoT) component. The chosen PoT algorithm is sequential AES, tuned for 1 proof per second. PoT is a measure against long-range attacks, and addresses unpredictability and dynamic availability issues. 
```

Comment:
```suggestion
In addition to the Proof-of-Space component described in the previous section, Dilithium is secured by a Proof-of-Time (PoT) component. The chosen PoT algorithm is sequential AES, tuned for 1 second per proof. PoT is a measure against long-range attacks, and addresses unpredictability and dynamic availability issues.

```
The extra  line is on purpose, to make the next block of text in a new paragraph